lib/index.coffee

```diff
@@ -118,7 +118,8 @@ module.exports = (opts) ->
       _.assign(_.omit(e, 'fields'), e.fields)
 
     ###*
-     * Sets `_url` property on content with single entry views
+     * Sets `_url` and `_urls` properties on content with single entry views
+     * `_url` takes the value `null` if the content type's custom path function returns multiple paths
```

Comment:
can you drop to a new line after `path function` so we keep it within our 80 char limit?